tests/testthat/test_civis_ml_utils.R

```diff
@@ -95,23 +114,16 @@ test_that("is_multitarget works", {
 test_that("get_predict_template_id returns correct template for train/predict version ", {
   m <- model_list[[1]]
   id <- m$job$fromTemplateId
-  ver <- CIVIS_ML_TEMPLATE_IDS[CIVIS_ML_TEMPLATE_IDS$id == id, "version"]
-  pred_id <- CIVIS_ML_TEMPLATE_IDS[CIVIS_ML_TEMPLATE_IDS$version == ver &
-                                   CIVIS_ML_TEMPLATE_IDS$name == "predict", "id"]
+  civis_ml_template_ids <- get_template_ids_all_versions()
+  ver <- civis_ml_template_ids[civis_ml_template_ids$id == id, "version"]
+  pred_id <- civis_ml_template_ids[civis_ml_template_ids$version == ver[1] &
+                                   civis_ml_template_ids$name == "prediction", "id"]
   expect_equal(get_predict_template_id(m), pred_id)
 
   fake_model <- list(job = list(fromTemplateId = 9112))
   expect_equal(get_predict_template_id(fake_model), 9113)
 })
 
-test_that("get_train_template_id reverts to last id if others not available", {
```

Comment:
I deleted this test b/c per @jacksonllee comment in PARO-741

Question: Can we assume the user will always have access to the latest version of CivisML?

Yes. All CivisML aliases for released versions of CivisML are public to all platform users.